src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -404,9 +400,11 @@ Environment variables:
 				if err != nil {
 					return fmt.Errorf("could not parse timeout duration %q: %v", timeout, err)
 				}
-				pachClient, err = client.NewOnUserMachine(false, !noPortForwarding, "user", client.WithDialTimeout(timeout))
+				// TODO(ys): metrics used to be force-disabled here, do we still want that?
+				pachClient, err = client.NewOnUserMachine("user", client.WithDialTimeout(timeout))
 			} else {
-				pachClient, err = client.NewOnUserMachine(false, !noPortForwarding, "user")
```

Comment:
@jdoliner metrics were disabled when creating a client for `pachctl version`. I'm not sure if this was a mistake, or by design. I'm assuming the former here, is that incorrect?

Reply:
Yeah, seems like a mistake to me.